devlib/module/gem5stats.py

```diff
@@ -107,14 +108,53 @@ def start_periodic_dump(self, delay_ns=0, period_ns=10000000):
     
     def match(self, keys, rois_labels, base_dump=0):
         '''
-        Tries to match the list of keys passed as parameter over the statistics
-        dumps covered by selected ROIs since ``base_dump``. Returns a dict 
-        indexed by key parameters containing a dict indexed by ROI labels 
-        containing an in-order list of records for the key under consideration
-        during the active intervals of the ROI.
-
-        Keys must match fields in gem5's statistics log file. Key example:
-            system.cluster0.cores0.power_model.static_power
+        Extract specific values from the statistics log file of gem5
+
+        :param keys: list of regex matched over the fields of the statistics 
```

Comment:
" a list of key name or regular expression patterns that will be matched in the fields of the statistics"